lib/mix/lib/mix/tasks/escript.build.ex

```diff
@@ -44,6 +44,9 @@ defmodule Mix.Tasks.Escript.Build do
       Defaults to app name. Set it to `nil` if no application should
       be started.
 
+    * `:strip_beam` - if `true` strip BEAM code in the escript to remove
+      debug information (chunk `abstract_code`). Defaults to `true`.
```

Comment:
"to remove unnecessary chunks at runtime, such as debug information and documentation. Defaults to `true`."